Bump fileset version

lukyth opened the pull request:
The current fileset version use minimatch@2.0.10 which might cause a RegExp DoS issue.

Update fileset to the latest version fix this warning.

This will resolve #6

lukyth committed:
Bump fileset version

The current fileset version use minimatch@2.0.10 which might cause a
RegExp DoS issue.

Update fileset to the latest version fix this warning.

yahoocla commented:
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

yahoocla commented:
CLA is valid!

lukyth commented:
I've tested it and found out that the test was failed even with the old `fileset` version. Probably need to fix the test first.

jmfrancois commented:
Yes please @lukyth fix the tests first

montogeek commented:
Test are fixed on https://github.com/istanbuljs/istanbul-api/pull/9

bcoe commented:
@lukyth mind rebasing with master? sounds like tests should start passing after?

lukyth committed:
Fix some test case

There're some test case which try to check if node_modules/adder.js is
exist, but it's not, so I fix those test cases.

lukyth commented:
@bcoe Done! 😄

SimenB commented:
@bcoe Can this get merged and released? Really annoying that deprecation warning from Jest on almost every single yarn command

bcoe commented:
@SimenB 👍 bumping to the top of my list.

bcoe commented:
@lukyth thanks a ton for the contribution, I'll have a candidate release out soon and will flag the Jest folks. FYI, here's why those tests broke:

https://github.com/istanbuljs/istanbul-api/commit/7659ef9da612fadc7e3a4b6c17f59afbd2533e3f

`node_modules` is excluded by default in fileset now, for the time being I removed this behavior since we already do this.

bcoe commented:
@lukyth would love the extra set of eyes testing, you should now be able to run:

`npm cache clear; npm i istanbul-api@next` to experiment with a candidate release that has the upgraded fileset dependency.
